appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -372,6 +378,7 @@ public UserResource toResource(UserDTO user)
 					addWeekActivityOverviewsLink(userResource);
 					addNewDeviceRequestLink(userResource);
 					addAppActivityLink(userResource);
+					pinResetRequestController.addLinks(userResource);
```

Comment:
This way does not follow the pattern. Other controllers are just exposing internal/public methods to get the link, so for consistency, should we not follow the same pattern here?